ch.sbb.scion.rcp.microfrontend.app.demo/META-INF/MANIFEST.MF

```diff
@@ -16,7 +16,10 @@ Require-Bundle: javax.inject;bundle-version="0.0.0",
  org.eclipse.core.runtime;bundle-version="0.0.0",
  ch.sbb.scion.rcp.microfrontend;bundle-version="1.0.0",
  org.eclipse.e4.core.commands,
- com.google.gson
+ com.google.gson,
+ org.eclipse.core.databinding;bundle-version="1.11.0",
```

Comment:
Do we specify the plug-in versions in the manifest?

Reply:
Proposal: We do a cleanup here in a later stage (move to import-package with correct version ranges).